rust/src/string.rs

```diff
@@ -87,34 +102,11 @@ impl BnString {
         mem::forget(value);
         res
     }
-
-    pub fn as_str(&self) -> &str {
```

Comment:
I guess the idea is that these methods are available via deref to `CStr` so there's no need to wrap them?